modules/serverless-tech-preview-features.adoc

```diff
@@ -95,11 +100,6 @@ The following table provides information about which {ServerlessProductName} fea
 |GA
 |GA
 
-|TLS for internal traffic
```

Comment:
Why did you remove the "TLS for internal traffic" entry?

Reply:
Here in this JIRA: https://issues.redhat.com/browse/SRVKS-1201?src=confmacro

It says it is a **removed functionality** so added it in the deprecated table.